desktop/conf.dist/hue.ini

```diff
@@ -84,6 +84,9 @@
   # Execute this script to produce the SSL password. This will be used when `ssl_password` is not set.
   ## ssl_password_script=
 
+  # Django sets X-Frame-Options HTTP header to this value. Default is 'SAMEORIGIN'.
+  ## django_x_frame_options=DENY
+
   # List of allowed and disallowed ciphers in cipher list format.
```

Comment:
Could you also put the same in https://github.com/cloudera/hue/blob/master/desktop/conf/pseudo-distributed.ini.tmpl ?
(used when starting in dev mode)